webf/lib/src/css/style_declaration.dart

```diff
@@ -384,7 +386,7 @@ class CSSStyleDeclaration with IterableMixin {
 
     // From style sheet mark the property important as false.
     if (isImportant == false) {
-      _sheetStyle[propertyName] = value;
+      _sheetStyle[propertyName] = normalizedValue;
```

Comment:
Why not use the formatted value？

Reply:
you mean used trimmed value that it will block multiple-blank value? such as pseudo element blank value?

Reply:
my bad